client/package.json

```diff
@@ -20,24 +23,46 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-runtime": "^6.23.0",
+    "chart.js": "^2.6.0",
     "chartjs": "^0.3.24",
     "chartkick": "^2.2.4",
     "es5-shim": "^4.5.9",
     "expose-loader": "^0.7.3",
     "imports-loader": "^0.7.1",
     "js-yaml": "^3.8.2",
     "lodash": "^4.17.4",
+    "prop-types": "^15.5.10",
     "react": "^15.5.4",
     "react-chartkick": "^0.1.3",
     "react-dom": "^15.5.4",
     "react-on-rails": "8.0.0",
+    "redux": "^3.7.2",
```

Comment:
it might be pre-emptive, my trigger finger just yarned it in xD

@abrophy  has checked out this working branch and so we're going to merge it and continue work on the `react-moment-dashboards-chart-control` branch.